source/databricks/calculation_engine/package/calculation/calculation_results.py

```diff
@@ -62,6 +62,9 @@ class TotalMonthlyAmountsContainer:
 class BasisDataContainer:
     metering_point_periods: DataFrame
     time_series_points: DataFrame
+    charge_master_data: DataFrame
+    charge_prices: DataFrame
+    charge_links: DataFrame
```

Comment:
Can't these be None?
```suggestion
    charge_master_data: DataFrame | None
    charge_prices: DataFrame | None
    charge_links: DataFrame | None
```